benchmark/src/main/scala/spire/benchmark/ZigguratBenchmarks.scala

```diff
@@ -1,15 +1,8 @@
 /**
- * **********************************************************************\
- * * Project                                                              **
- * *       ______  ______   __    ______    ____                          **
- * *      / ____/ / __  /  / /   / __  /   / __/     (c) 2011-2014        **
- * *     / /__   / /_/ /  / /   / /_/ /   / /_                            **
- * *    /___  / / ____/  / /   / __  /   / __/   Erik Osheim, Tom Switzer **
- * *   ____/ / / /      / /   / / | |   / /__                             **
- * *  /_____/ /_/      /_/   /_/  |_|  /____/     All rights reserved.    **
- * *                                                                      **
- * *      Redistribution and use permitted under the MIT license.         **
- * *                                                                      **
+ * **********************************************************************\ * Project ** * ______ ______ __ ______ ____
+ * ** * / ____/ / __ / / / / __ / / __/ (c) 2011-2014 ** * / /__ / /_/ / / / / /_/ / / /_ ** * /___ / / ____/ / / / __ /
+ * / __/ Erik Osheim, Tom Switzer ** * ____/ / / / / / / / | | / /__ ** * /_____/ /_/ /_/ /_/ |_| /____/ All rights
+ * reserved. ** * ** * Redistribution and use permitted under the MIT license. ** * **
  * \***********************************************************************
  */
```

Comment:
Oh no 🤦

Reply:
follow up in #1086

Reply:
I think you can indicate a block doesn't need formatting
https://scalameta.org/scalafmt/docs/configuration.html#-format-off

Reply:
Yes, but I've never seen anybody use that for their headers. Maybe this situation is different, but I'd prefer a better fix if possible.